### PR TITLE
Navbar footer fix error

### PR DESCRIPTION
**Pull request by LaserHuy**

### CODE REVIEW

**Review by Benja-san — APPROVED** [review]

File: src/View/Item/navbar.html.twig
Comment (yazidsef) [inline src/View/Item/navbar.html.twig:1]: please make this foldr on components

**Review by yazidsef — CHANGES_REQUESTED** [review]
onlynavbar put it on components

File: src/View/Item/navbar.html.twig
Comment (LaserHuy) [reply src/View/Item/navbar.html.twig:1]: already Done

**Review by yazidsef — APPROVED** [review]
okay for everything

**Pull request merged**